src/config/interfaces/jwt.interface.ts

```diff
@@ -0,0 +1,11 @@
+export interface IJwtToken {
```

Comment:
Why you name the interface IJwtToken 

The first "I" it mean Interface ? 
I don't think it neccessary.